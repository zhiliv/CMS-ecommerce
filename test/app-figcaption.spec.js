import { shallowMount } from '@vue/test-utils'
import Container from '@/components/app/figcaption/figcaption.vue'

describe('app-figcaption.js', () => {
  let cmp

  beforeEach(() => {
    cmp = shallowMount(Container, {
      // Создать поверхностный экземпляр компонента
      propsData: {
        classes: 'simple',
        styles: 'background: white',
        id: 'indent',
      },
    })
  })
  it('Содержимое html для app-figcaption ', () => {
    expect(cmp.element).toMatchSnapshot()
  })
})
