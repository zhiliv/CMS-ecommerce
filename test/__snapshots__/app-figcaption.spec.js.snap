// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app-figcaption.js Содержимое html для app-figcaption  1`] = `
<figcaption
  class="figure-caption simple"
  id="indent"
/>
`;
